skyvern/library/ai_locator.py

```
@@ -0,0 +1,123 @@
+                    xpath = await self._page_ai.ai_locate_element(prompt=self._prompt)
+                    if not xpath:
+                        raise ValueError(f"AI failed to locate element with prompt: {self._prompt}")
+
+                    self._resolved_locator = self._page.locator(
+                        xpath if xpath.startswith(("xpath=", "css=", "text=", "role=", "id=")) else f"xpath={xpath}"
+                    )
+                except Exception as e:
+                    if self._selector and not self._try_selector_first:
+                        self._resolved_locator = self._page.locator(self._selector, **self._selector_kwargs)
+                    else:
+                        raise e
```

Ruff (0.14.5)

Use `raise` without specifying exception name

Remove exception name

(TRY201)